### ISSUE TITLE
Switching keyboard layouts for different languages (e.g. Russian)

### ISSUE THREAD

**Issue by AlexDaniel** [yuliskov/SmartTubeNext#1233]
The bulit-in keyboard only has QWERTY layout and there does not seem to be any way to add more layouts. This is a problem for languages like Russian where the entire alphabet is different and requires another keyboard layout. Please add a way to configure layouts and switch between them (when using Search).

**Comment by ksa-real** [comment]
How have you solved it? I have a US version Firestick 4k Max, and it seems there is no way to configure Russian layout.

**Comment by AlexDaniel** [comment]
@ksa-real there has to be. Somewhere in global settings of the device itself (not smarttube settings).

**Comment by ksa-real** [comment]
Yeah, I guess bad luck. Likely the list is region locked. In the US it doesn't offer the Russian option.
